Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,35 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import { ClearButton, ZeroButton } from './components/ButtonComponents/ActionButton';
+import OperatorButton from './components/ButtonComponents/OperatorButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+
+const calcButtons = {
+  operators: ['÷', '×', '−', '+', '='],
+  numbers: [7, 8, 9, 4, 5, 6, 1, 2, 3,],
+  action1: 'clear',
+  action2: 0,
+};
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="app">
+            <CalculatorDisplay />
+            <div className="all-buttons">
+                <div className="action-number-buttons">
+                <ClearButton action={calcButtons} />
+                <div className="number-buttons">
+                  {calcButtons.numbers.map(number => (
+                  <NumberButton numberButtonStyle="number-item" key={number.toString()} text={number} />
```

Comment:
is there a reason why for number.toString() instead of just number ?

Reply:
@cbtsao47 ah no reason, I forgot to remove toString(), I was debugging this to get key to work